webrtc-stats.html

```diff
@@ -1189,6 +1189,8 @@
              boolean             ended;
              boolean             detached;
              DOMString           kind;
+             DOMHighResTimeStamp estimatedPlayoutTimestamp;
+             DOMHighResTimeStamp estimatedCaptureTimestamp;
```

Comment:
I don't think we should have "estimated" in the name as long as it's not possible to get a better value.